script.js

```diff
@@ -0,0 +1,2 @@
+console.log('hello Javascript');
```

Comment:
Why did you add this console log?
```suggestion
console.log('hello Pull Request');
```